scijava-ops-api/pom.xml

```diff
@@ -12,7 +12,7 @@
 	<artifactId>scijava-ops-api</artifactId>
 
 	<name>SciJava Ops API</name>
-	<description>SciJava Operations API: Outward-facing Interfaces used by the SciJava Operations framework.</description>
```

Comment:
I like invoking "user" connotations here, what do you think about "The user interface for the SciJava Ops framework"?

Reply:
Or something similar

Reply:
The term "user interface" implies GUI, so I'd rather not. Something with "application programming interface" or just "API" would be OK with me though. GitHub won't show me what I actually changed the description to here, which is frustrating.

Reply:
"User API"?